sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosItemIdEncodingTest.java

```diff
@@ -616,6 +743,8 @@ public TestScenario(
 
         public TestScenarioExpectations Gateway;
 
+        public TestScenarioExpectations ComputeGateway;
```

Comment:
nit: variable name starts with lowercase

Reply:
Fixed in next iteration.